src/lib.rs

```diff
@@ -45,16 +50,37 @@ impl AsRef<str> for MessageContent {
     }
 }
 
+impl Display for MessageContent {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
+
+// This deserialization implementation is going to get used only when building
+// the initial SQLite database. It will first deserialize its input as a string,
+// then escape the HTML entities, then replace the (now escaped; see regex
+// below) instances of emote tags with the equivalent HTML `img` element.
+//
+// Discord emote tags are of the form `<a:FrankerZ:12345678>`. If the `a`
```

Comment:
Good that you are documenting these quirks in the code SeemsZ

Reply:
ComfWee